### ISSUE TITLE
Show path of default jdk as well in the notification

### ISSUE THREAD

**Issue by yuyully** [microsoft/build-server-for-gradle#196]
Gradle for java: v3.17.0

When default sdk is not compatible with current gradle version, there is a notification as below, the message should also show what default sdk version is detected.

Repro steps:
1. Make sure there is JDK 8 in your local machine and JAVA_HOME is not JDK 8.
2. Don't set gradle import java version in vscode settings
3. Clone and open https://github.com/lenskit/lenskit.git

![Image](https://github.com/user-attachments/assets/2110efc0-15e0-4c50-a3a3-80ab6ac7915e)

**Comment by jdneo** [comment]
@Tanish-Ranjan

Do you remember that the default jdk is the jdk used to launch the gradle server, right?

**Comment by Tanish-Ranjan** [comment]
@jdneo Yes if the default jdk is compatible, it will be used to launch the BuildServer otherwise we try to find a compatible jdk.

**Comment by jdneo** [comment]
Ok

Convert the request to 'Show path of default jdk as well in the notification'
